src/promptflow/promptflow/executor/_tool_resolver.py

```diff
@@ -240,6 +247,20 @@ def _resolve_llm_node(self, node: Node, convert_input_types=False) -> ResolvedTo
         api_func = partial(api_func, **{prompt_tpl_param_name: prompt_tpl}) if prompt_tpl_param_name else api_func
         return ResolvedTool(updated_node, tool, api_func, init_args)
 
+    def _resolve_replay_node(self, node: Node, convert_input_types=False) -> ResolvedTool:
```

Comment:
It would be better to use mock to achieve the same target.

Reply:
Thx